observability/pillars/dashboard.md

```diff
@@ -0,0 +1,58 @@
+# Dashboard and Visualization
```

Comment:
Suggestion: Use the word Dashboard or Visualization; not both. 
Usually dashboards are considered as a data visualization tool. It is not clear on whether we want to distinguish between the two and why? 

Your thoughts?

Reply:
Dashboards could technically couldl also be a concise way of presenting information, imho

Reply:
Changed to "Dashboard" since there's no major mention of other visualization